CONTRIBUTING.md

```diff
@@ -1,10 +1,8 @@
-### :bug: [How to report a bug](http://polite.technology/reportabug.html)
+## Bug Reporting :bug:
 
-## Please note
+One of the most important things you can do is report bugs. Please reference [how to report a bug](http://polite.technology/reportabug.html) and follow the issue templates when adding new bugs.
 
-The high-level structure of Turf is undergoing discussion at [#1428](https://github.com/Turfjs/turf/issues/1428). Currently (June 2018), there is a partial conversion to Typescript, and the contribution documentation does not completely reflect the current status. 
```

Comment:
Removed this. It seems like we're relatively stable on half typescript half javascript for the time being.